### PR TITLE
Integrate first style sketch

### PR DESCRIPTION
**Pull request by LukasKalbertodt**
Here is my first sketch of the website style.

### CODE REVIEW

**Comment by JulianKniephoff** [comment]
Dude, more atomic commits. ;)

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  <div class="site-frame">
+    <div class="header-top">
+      <div class="header-title">
+        Neue Konzepte der Informatik
+      </div>
+      <div class="header-user">
+        <% if signed_in? %>
+          <%= link_to 'Abmelden', logout_path, method: :delete %>
+        <% else %>
+          <!-- <%= link_to 'Anmelden', login_path %> # -->
```
Comment (JulianKniephoff) [inline app/views/layouts/_header.html.erb]: Commenting out erb code should be done using
```erb
<%# link_to ... %>
```
syntax

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          <!-- <button>Login</button> -->
+          <%= button_to "Login", login_path, class: "header-login-button", method: "get" %>
```
Comment (JulianKniephoff) [inline app/views/layouts/_header.html.erb]: We have to investigate why `method: :get` does not work, since it is actually "the way"(TM).

Comment (LukasKalbertodt) [inline app/views/layouts/_header.html.erb]: It does work. We tested `:GET` which doesn't.

**Comment by JulianKniephoff** [comment]
We want to use `'` instead of `"` for strings unless interpolation is needed.

**Pull request merged**